resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/MountVolumesFeatureStep.scala

```diff
@@ -120,6 +122,20 @@ private[spark] class MountVolumesFeatureStep(conf: KubernetesConf)
   override def getAdditionalKubernetesResources(): Seq[HasMetadata] = {
     additionalResources.toSeq
   }
+
+  private def checkPVCClaimNameWhenMultiExecutors(claimName: String): Unit = {
```

Comment:
`checkPVCClaimNameWhenMultiExecutors` -> `checkPVCClaimName` because this PR checks all cases instead of `WhenMultiExecutors` and doesn't raise exception for single instance case.

Reply:
Change it as `checkPVCClaimName` :)